pyradiosky/skymodel.py

```diff
@@ -70,6 +72,46 @@ class LunarTopo:
 #    Convert stokes and coherency to Astropy quantities.
 
 
+class TelescopeLocationParameter(UVParameter):
```

Comment:
What effect does this have?

Reply:
This fixes a problem with comparing EarthLocation objects. If I just use the normal UVParameter class the check fails because when it gets the value it comes back as an array (for the x,y,z positions) but indexing into it causes an error (because it's a single Quantity). I thought about trying to fix it in pyuvdata somehow but then decided that the right way to handle it was subclassing UVParameter for this special case. And since it's really a very specific issue for SkyModel, the subclassing belonged here. This could change, I suppose, if we decided to make the telescope location in pyuvdata be an EarthLocation object, but I don't see that happening anytime soon.